topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -109,7 +109,7 @@ private Value(String key, Type<? extends T> type, boolean required) {
     private static final Map<String, Value<?>> CONFIG_VALUES = new HashMap<>();
 
     /** A comma-separated list of key=value pairs for selecting Resources that describe topics. */
-    public static final Value<LabelPredicate> LABELS = new Value<>(TC_RESOURCE_LABELS, LABEL_PREDICATE, "strimzi.io/kind=topic");
+    public static final Value<LabelPredicate> LABELS = new Value<>(TC_RESOURCE_LABELS, LABEL_PREDICATE, "strimzi.io/cluster=my-cluster");
```

Comment:
Do I read this correctly as a default value? If yes, then I think it should be empty by default (i.e. watch for all `KafkaTopic` resources).